components/capi/ctest/test.c

```diff
@@ -0,0 +1,36 @@
+// This file is part of ICU4X. For terms of use, please see the file
```

Comment:
I'm not sure if the intention is a quick test or something more permanent, but I'm guessing we'll want example code for all of this eventually. Maybe add an examples subdir and move this to there?

Reply:
This is a quick test, Proper Examples can be worked on later (especially once we have more of an API surface)

Reply:
Please do consider moving this to components/capi/examples/pluralrules. We're going to want something like that eventually, and putting it there now could help us think about to structure things in the future.

Reply:
I didn't want to use `examples` because it means something specific to `cargo`, but if we use a second folder (or don't use Rust files) it won't matter